composer.json

```diff
@@ -4,8 +4,10 @@
     "type": "psalm-plugin",
     "require": {
         "barryvdh/laravel-ide-helper": "^2.6",
+        "laravel/framework": "^5.5 || ^6.0",
         "vimeo/psalm": "^3.8.2",
-        "orchestra/testbench": "^3.5 || ^4.0 || ^5.0"
+        "orchestra/testbench": "^3.5 || ^4.0 || ^5.0",
+        "psr/event-dispatcher": "^1.0"
```

Comment:
When trying to run tests, the existing code was throwing this class not found exception in `FakeMetaCommand.php`. I'm not sure what is relying on this

Reply:
hm, this is probably related to https://github.com/psalm/psalm-plugin-laravel/issues/24